library/Director/Resolver/CommandUsage.php

```diff
@@ -44,6 +44,7 @@ public function getLinks()
             'host'         => ['check_command', 'event_command'],
             'service'      => ['check_command', 'event_command'],
             'notification' => ['command'],
+            'command'      => ['command'],
```

Comment:
```suggestion
            'command'      => [],
```

This could be an empty array, as this is basically used to build the command id column for the table to filter the objects using the corresponding command.

Reply:
I think I added that because a command can inherit from a command but after so much time I am not sure anymore